frame/democracy/Cargo.toml

```diff
@@ -27,9 +27,9 @@ serde = { version = "1.0.136", optional = true, features = ["derive"] }
 sp-io = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.18" }
 sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.18" }
 sp-std = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.18" }
+orml-tokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", rev = "2b5d4ce1d08fb54c0007c2055653892d2c93a92e", default-features = false }
```

Comment:
I think orml-tokens only needs to be a dev-dependency right? Or are we using it in the benches? Then we can still put it behind the benchmarking flag

Reply:
There was a build error with orml-tokens under dev-dependencies, that fixed it, moved it to dev-dependencies , its fixed now

Reply:
it is used in the test runtime

Reply:
fixed this, moved to dev-dependency and updated feautures